File: Azure/AdventureWorksOrder/AdventureWorksOrder.Model/Custom/SalesOrderDetailItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdventureWorksOrder.Model.Custom
{
    public class SalesOrderDetailItem
    {
        public string CarrierTrackingNumber { get; set; }

        public decimal LineTotal { get; set; }

        public DateTime ModifiedDate { get; set; }

        public short OrderQty { get; set; }

        public EntityInfo Product { get; set; }

        public int SalesOrderDetailId { get; set; }

        public int SalesOrderId { get; set; }

        public decimal UnitPrice { get; set; }

        public decimal UnitPriceDiscount { get; set; }
    }
}

File: Azure/AdventureWorksOrder/Web/src/model/sales-order-detail-item.ts
import { EntityInfo } from "./entity-info";

export interface SalesOrderDetailItem {
  carrierTrackingNumber?: any;
  lineTotal: number;
  modifiedDate: string;
  orderQty: number;
  product: EntityInfo;
  salesOrderDetailId: number;
  salesOrderId: number;
  unitPrice: number;
  unitPriceDiscount: number;
}
